FreeRTOS/FreeRTOS-Kernel
# Update third party ports readme

Description
-----------
Update third party ports readme

Test Steps
-----------
N/A (no code change)..

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
